docs/exts/docs_build/lint_checks.py

```diff
@@ -268,7 +268,7 @@ def check_enforce_code_block() -> List[DocBuildError]:
 def find_example_dags(provider_dir):
     system_tests_dir = provider_dir.replace(f"{ROOT_PACKAGE_DIR}/", "")
     yield from glob(f"{provider_dir}/**/*example_dags", recursive=True)
-    yield from glob(f"{ROOT_PROJECT_DIR}/tests/system/{system_tests_dir}/*/", recursive=True)
+    yield from glob(f"{ROOT_PROJECT_DIR}/tests/system/{system_tests_dir}/**/example_dags", recursive=True)
```

Comment:
I see that you have moved example_slack.py to :
![image](https://user-images.githubusercontent.com/8532066/171565933-5da64718-3b09-4d1c-ae58-6e179830e92c.png)

It shouldn't be inside example_dags directory - it can either directly under slack/ or under another directory that groups several examples into one. If some checks are failing and it's related to that - let me know so we can fix the check to match different kind of paths (but it shouldn't - take a look to ``elasticsearch`` system tests which have similar structure).

Reply:
After some debugging I see that this lint check is not entirely correct (that's why it could fail for you as well). The path to the examples dags in the new design should be:
```
yield from glob(f"{ROOT_PROJECT_DIR}/tests/system/{system_tests_dir}**/*", recursive=True)
```

Reply:
@rbiegacz ?